vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -1390,6 +1390,80 @@ public Grid() {
         this(50);
     }
 
+    /**
+     * Creates a new grid using the given generic {@code DataProvider}.
+     *
+     * @param dataProvider
+     *            the data provider, not {@code null}
+     *
+     */
+    public Grid(DataProvider<T, Void> dataProvider) {
+        this(50);
```

Comment:
No need to duplicate this constant everywhere, we could just reuse the constructor without any arguments, which initializes with a page size of `50` by default.